plat/hisilicon/hikey960/hikey960_pm.c

```diff
@@ -161,34 +122,38 @@ static void __dead2 hikey960_system_reset(void)
 int hikey960_validate_power_state(unsigned int power_state,
 			       psci_power_state_t *req_state)
 {
-	unsigned int state_id;
+	int pstate = psci_get_pstate_type(power_state);
+	int pwr_lvl = psci_get_pstate_pwrlvl(power_state);
```

Comment:
Static analysis failure:
Incorrect type of operands at lines 126, 131 and 140.
Make pwr_lvl unsigned int would solve.

Similarly for pstate - change to unsigned int to avoid same static check fail

Reply:
Hi @davidcunado-arm, have uploaded new patches for fixing these two static analysis.

Please let me know if still have issue. Thanks.